opendistro-for-elasticsearch/opendistro-build
# [HELM] Kibana extra volumes

*Description of changes:*
User can define extra volumes for Kibana

Test Results:

* Deploy using the following chart values :
```
kibana:
  extraVolumes:
    - name: my-ca-trust
      hostPath:
        path: "/etc/ssl/certs"

  extraVolumeMounts:
    - name: my-ca-trust
      mountPath: /usr/share/kibana/config/certs
      readOnly: true
```
* Check the volumes on the Kibana pod

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
